### ISSUE TITLE
GitHub PR Workflow Check: Talawa Admin app starts `(Docker)`

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-admin#2402]
**Describe the bug**

The GitHub PR workflow does not check whether the Talawa Admin app can start in Docker

**To Reproduce**

1. Make a change to the repo that makes the Talawa Admin app fail to start in Docker
1. Trigger the workflow to run as part of a PR
1. The workflow **_passes_**

and as expected:

1. Make a change to the repo that makes the Talawa Admin app start in Docker
1. Trigger the workflow to run as part of a PR
1. The workflow passes

**Expected behavior**

1. Make a change to the repo that makes the Talawa Admin app fail to start in Docker
1. Trigger the workflow to run as part of a PR
1. The workflow **_fails_**

and as expected:

1. Make a change to the repo that makes the Talawa Admin app start in Docker
1. Trigger the workflow to run as part of a PR
1. The workflow passes

We now have a test data that can be loaded that resides in the repo. This should be used as part of the test.

**Actual behavior**

- The Talawa Admin app isn't started and tested in the PR workflow

**Screenshots**

- N/A

**Additional details**

1. It must verify starting with Docker
1. Use this issue for reference:
    1. https://github.com/PalisadoesFoundation/talawa-api/issues/1343

**Potential internship candidates**

- Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by VanshikaSabharwal** [comment]
@palisadoes i can work on this issue. Kindly assign me this issue.

My Approach :- i would update the GitHub PR workflow to include a Docker start check for the Talawa Admin app. Integrate a test step that attempts to start the app in Docker and fails the workflow if it doesn’t start successfully. Use the existing test data in the repo to verify that the app loads correctly in Docker as part of the CI/CD process.

**Comment by palisadoes** [comment]
During the week of November 11, 2024 we will start a code freeze on the `develop` branches in Talawa, Talawa Admin and Talawa-API. 

We have completed a project to convert the Talawa-API backend to use PostgreSQL. Work will then begin with us merging code in the `develop` branches to a new `develop-postrgres` branch in each repository.

Planning activities for this will be managed in our #talawa-projects slack channel. A GitHub project will be created to track specially labeled issues. We completed a similar exercise last year using a similar methodology. 

Starting November 12, California time no new PRs will be accepted against the `develop` branch. They must be applied to the `develop-postrgres` branch. 

There are some GSoC project features that will need to be merged into develop. These will be the only exceptions.

This activity and the post GSoC 2024 start date was announced in our #general Slack channel last month as a pinned post.
